docs/ops/sort/MulticlassNMS_8.md

```diff
@@ -0,0 +1,160 @@
+    * *none* - do not guarantee the order.
+  * **Type**: `string`
+  * **Default value**: `none`
+  * **Required**: *No*
+
+* *sort_result_across_batch*
+
+  * **Description**: *sort_result_across_batch* is a flag that specifies whenever it is necessary to sort selected boxes across batches or not.
+  * **Range of values**: true or false
+    * *true* - sort selected boxes across batches.
+    * *false* - do not sort selected boxes across batches (boxes are sorted per batch element).
+  * **Type**: boolean
+  * **Default value**: false
+  * **Required**: *No*
+
+* *output_type*
+
+  * **Description**: the tensor type of outputs `selected_indices` and `valid_outputs`.
+  * **Range of values**: `i64` or `i32`
+  * **Type**: `string`
+  * **Default value**: `i64`
+  * **Required**: *No*
+
+* *iou_threshold*
+
+  * **Description**: intersection over union threshold.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+* *score_threshold*
+
+  * **Description**: minimum score to consider box for the processing.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+* *nms_top_k*
+
+  * **Description**: maximum number of boxes to be selected per class.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *keep_top_k*
+
+  * **Description**: maximum number of boxes to be selected per batch element.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes.
+  * **Required**: *No*
+
+* *nms_eta*
+
+  * **Description**: eta parameter for adaptive NMS.
+  * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+  * **Type**: `float`
+  * **Default value**: `1.0`
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. The box coordinates are layout as `[xmin, ymin, xmax, ymax]`. **Required.**
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. **Required.**
+
+**Outputs**:
+
+*   **1**: `selected_outputs` - tensor of type *T_THRESHOLDS* and shape `[number of selected boxes, 6]` containing the selected boxes with score and class as tuples `[class_id, box_score, xmin, ymin, xmax, ymax]`.
+
+*   **2**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, 1]` the selected indices in the flattened `boxes`, which are absolute values cross batches. Therefore possible valid values are in the range [0, num_batches * num_boxes -1].
```

Comment:
Wrap `[0, num_batches * num_boxes -1]` with "`" characters and add one space character before the "1".

Reply:
okay. 
another question is about `[number of selected boxes, 1]`. Do you think if `[number of selected boxes]` is better? @lazarevevgeny

Reply:
Yes. I think that trailing `1` looks awkward.